src/lsm/groove.zig

```diff
@@ -926,9 +924,13 @@ pub fn GrooveType(
             inline for (std.meta.fields(IndexTrees)) |field| {
```

Comment:
I was originally thinking that we now need to assert that `id != 0` at the LSM layer, but the id tree is handled separately from the secondary indexes, so nevermind!